<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkingPapers</class>
 <widget class="QDialog" name="MarkingPapers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>580</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBPrivious">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>582</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Privious</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPageNo">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>590</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBQuestions">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1111</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="fQuestion1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="fQuestion2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="fQuestion3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>1110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="fQuestion4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>1110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="fQuestion5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>1110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBNext">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>582</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBHeader">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
